src/components/FilterList.jsx

```diff
@@ -154,13 +154,17 @@ const FilterList = ({
   /*************************/
 
   const [apiEndpoint, setApiEndpoint] = useState(
-    () => (defaultApiEndpoint = buildDefaultEndPoint())
+    () => (defaultApiEndpoint = buildDefaultEndPoint(includeDateFilter))
   );
```

Comment:
This was defaulting every application to having a date. Passing in the prop of includeDateFilter resolves that.